### PR TITLE
Rename DatasetGroup to ResourceGroup

### PR DESCRIPTION
**Pull request by RobNicholsGDS**
Updated relative paths so that they work when called via another file.

Also correct some typo mistakes:
- misspelling of description
- repetition of type in dataService theme
- supplierIdentifier description miss labelled.

### CODE REVIEW

**Comment by JFreeland3931** [comment]
Reviewed and validated as requested, thanks

File: schema/data_marketplace_resource_group_schema.json [hunk 1]
```diff
@@ -89,11 +89,12 @@
       "maxLength": 250
     },
     "member": {
```
[the remark this answers is not in the document]
Comment (RobNicholsGDS) [reply schema/data_marketplace_resource_group_schema.json:91]: Interesting that you have suggested that. I'm used to using plurals for methods/attributes that return arrays or collections so would normally do so here too.

However, if you look at the other attributes within DCAT that doesn't seem to be the protocol used. For example: _theme_; _keyword_; and _distribution_. All return arrays. All singular.

[the remark this answers is not in the document]
Comment (asmith-nhsx) [reply schema/data_marketplace_resource_group_schema.json:91]: Okay that makes sense, am I right in thinking "member" is an attribute we have added beyond the DCAT specification? But we should still be consistent with their conventions as you say.

[the remark this answers is not in the document]
Comment (RobNicholsGDS) [reply schema/data_marketplace_resource_group_schema.json:91]: Yes - it is a new attribute.

**Pull request merged**